src/axom/slam/Map.hpp

```diff
@@ -261,32 +270,98 @@ class Map : public StrPol,
     return IndirectionPolicy::getIndirection(m_data, setIndex);
   }
 
+  /**
+   * \brief Access the value associated with the given position in the set.
+   */
+  AXOM_HOST_DEVICE ConstValueType operator()(SetPosition setIdx) const
+  {
+    // TODO: validate that runtime stride is 1-D with value 1?
+    return value(setIdx, 0);
+  }
+
+  /// \overload
+  AXOM_HOST_DEVICE ValueType operator()(SetPosition setIdx)
+  {
+    return value(setIdx, 0);
+  }
+
   /**
    * \brief Access the value associated with the given position in the set and
    *        the component index.
    *
    * \pre `0 <= setIdx < size()`
-   * \pre `0 <= comp < numComp()`
+   * \pre `0 <= compIdx[i] < shape()[i]`
```

Comment:
Should `i` be `setIdx`?

Reply:
`i` represents the dimensionality of the stride. Updated docs to correspond to this.